djtriggers/models.py

```diff
@@ -57,7 +59,7 @@ class Trigger(models.Model):
 
     trigger_type = models.CharField(max_length=50, db_index=True)
     source = models.CharField(max_length=250, null=True, blank=True)
-    date_received = models.DateTimeField()
+    date_received = models.DateTimeField(default=datetime.now)
```

Comment:
I think you want this one instead: https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.DateField.auto_now_add

`datetime.now` is - as far as I can tell - a method object.